docs/source/class_basics.rst

```diff
@@ -77,6 +77,53 @@ to it explicitly using ``self``:
            a = self
            a.x = 1      # Error: 'x' not defined
 
+Class and instance variable annotations
+***************************************
+
+Mypy supports annotations for class and instance 
```

Comment:
This section doesn't flow well from the previous section due to different terminology. In the last section we talk about class and instance attributes and here we talk about class and instance variables, and here "class variable" means something different from "class attribute" in the previous section. Not sure what's the best way forward, but the two section should form a coherent discussion.